### PR TITLE
Th2 2344

### PR DESCRIPTION
**Pull request by alexey-suknatov-exactpro**

### CODE REVIEW

File: src/main/proto/th2_grpc_common/common.proto [hunk 1]
```diff
@@ -78,7 +78,8 @@ message ListValue {
 message Message {
   EventID parent_event_id = 3; // It maybe used to store event related to message life cycle
   MessageMetadata metadata = 1;
-  map<string, Value> fields = 2;
+  map<string, Value> fields = 2 [deprecated = true];
+  string json = 4;
```
Comment (ConnectDIY) [inline src/main/proto/th2_grpc_common/common.proto:82]: "json_body" expected in TH2-2344

Comment (OptimumOpium) [reply src/main/proto/th2_grpc_common/common.proto:82]: How about creating a message and add this field to the new message instead?
```
message Message {
  message Body {
    string json = 1;
    // or  bytes json = 1;
    bool zip = 2;
  }
  // ...
  Body body = 4;
}
```

Comment (alexey-suknatov-exactpro) [reply src/main/proto/th2_grpc_common/common.proto:82]: @ConnectDIY thought `json` is shorter but still recognizable

Comment (Nikita-Smirnov-Exactpro) [reply src/main/proto/th2_grpc_common/common.proto]: We haven't executed performance tests related to zipping. The zip field can be added in the future if necessary

**Comment by Nikita-Smirnov-Exactpro** [comment]
This change isn't actual

**Pull request closed without merging**